srcpkgs/sddm/template

```diff
@@ -28,9 +29,6 @@ conf_files="
  /etc/pam.d/sddm-greeter
  /etc/pam.d/sddm-autologin"
 
-# Both install /etc/dbus-1/system.d/org.freedesktop.DisplayManager.conf
-conflicts="lightdm>=0"
-
 pre_configure() {
 	touch -f /etc/login.defs
```

Comment:
this seems wrong, sddm uses login.defs to get important information, i think `shadow` should be added to `hostmakedepends`

```
CMake Warning at CMakeLists.txt:201 (message):
  Could not detect a valid UID_MIN (got '').  You should review your
  login.defs file.


CMake Warning at CMakeLists.txt:212 (message):
  Could not detect a valid UID_MAX (got '').  You should review your
  login.defs file.

```

Reply:
I agree, but also adding `shadow` to `hostmakedepends` does not work. It simply will hang at installing `elogind-devel`. I worked around this by using the login.defs in `srcpkgs/shadow/files`